recipes/airmessage/webview.js

```diff
@@ -1,8 +1,15 @@
-module.exports = () => {
-  // TODO: If your AirMessage service has unread messages, uncomment these lines to implement the logic for updating the badges
-  // const getMessages = () => {
-  //   // TODO: Insert your notification-finding code here
-  //   Ferdium.setBadge(0, 0);
-  // };
-  // Ferdium.loop(getMessages);
+module.exports = (Ferdium) => {
+  const getMessages = function getMessages() {
+    let count = 0;
+    let messageNameElements = document.getElementsByClassName("MuiTypography-body1");
+    if (messageNameElements.length > 0) {
+      for (i = 0; i < messageNameElements.length; i++) {
```

Comment:
Please fix this `for` loop. I would suggest making it more robust by using the automatic loops (in the like of `for (var element of messageNameElements)` which makes the test on the previous line unnecessary as well.